manifests/g/GiulioSorrentino/CBriscola/WPF/0.6.4/GiulioSorrentino.CBriscola.WPF.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GiulioSorrentino.CBriscola.WPF
+PackageVersion: 0.6.4
+InstallerType: inno
+Installers:
+- Architecture: x64

```

Comment:
```suggestion
- Architecture: arm64
```

Reply:
It's valid both for arm64 and x64, it's in IL and it's compiled when it's started.
It should work even on x86 and arm32

Reply:
I'm sorry, x86 and arm32 are blocked by innosetup

Reply:
Oh, I misunderstood.